pipelined/bevy_pbr2/src/render/mod.rs

```diff
@@ -250,7 +262,21 @@ pub fn queue_meshes(
         });
 
         let draw_pbr = draw_functions.read().get_id::<DrawPbr>().unwrap();
-        for i in 0..extracted_meshes.meshes.len() {
+        for (i, mesh) in extracted_meshes.meshes.iter_mut().enumerate() {
+            if mesh.material_bind_group_id.is_none() {
+                let material_bind_group = BindGroupBuilder::default()
+                    .add_binding(
+                        0,
+                        RenderResourceBinding::Buffer {
+                            buffer: mesh.material_buffer,
+                            range: 0..StandardMaterialUniformData::std140_size_static() as u64,
+                        },
+                    )
+                    .finish();
+                render_resources.create_bind_group(layout.bind_group(2).id, &material_bind_group);
```

Comment:
This will result in building / (attempting) to create redundant bind groups. create_bind_group checks for existence (which is detected using a hash of the bind group), so we won't _actually_ create extra bind groups. But this will incur overhead. It would be better to use a cache like we do for sprite textures.

Reply:
Done.